src/plugins/intel_gpu/tests/functional/shared_tests_instances/behavior/plugin/configuration_tests.cpp

```diff
@@ -89,8 +89,6 @@ namespace {
 
     auto auto_batch_inconfigs = []() {
         return std::vector<std::map<std::string, std::string>>{
-            {{CONFIG_KEY(AUTO_BATCH_DEVICE_CONFIG), CommonTestUtils::DEVICE_GPU},
-             {CONFIG_KEY(AUTO_BATCH_TIMEOUT), "-1"}},
             {{CONFIG_KEY(AUTO_BATCH_DEVICE_CONFIG), CommonTestUtils::DEVICE_GPU},
```

Comment:
something minor but better to make sure, if with legacy api LoadNetwork(***, {CONFIG_KEY(AUTO_BATCH_TIMEOUT), "-1"}), batch can convert this string "-1" to unsigned int using *as* method? any chance it will throw exception?